doc/changelog/02-specification-language/14750-succeed.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Added :cmd:`Succeed`, which behaves like a doubled :cmd:`Fail` but keeps error messages intact
```

Comment:
No need to repeat `Added`.

```suggestion
  :cmd:`Succeed` control command, which behaves like a doubled :cmd:`Fail` but keeps error messages intact
```

Reply:
```suggestion
  :cmd:`Succeed`, a :n:`@control_command`that verifies that the given :n:`@sentence` succeeds without changing the proof state
```

The "doubled Fail but keeps error messages intact" is rather unclear.  "double Fail" seems like the moral equivalent of a double negative, which English avoids.